velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -141,6 +142,22 @@ Writer::Writer(
   } else {
     flushPolicy_ = std::make_unique<DefaultFlushPolicy>();
   }
+
+  if (options.schema) {
```

Comment:
What happens if schema is not specified? What will be the schema of the file?

Reply:
schema will be inferred from the record batch below
`arrowContext_->schema = recordBatch->schema();`